vda5050_msgs/msg/OrderState.msg

```diff
@@ -65,7 +65,7 @@ string operating_mode                                # Enum {AUTOMATIC, SEMIAUTO
 
 vda5050_msgs/Error[] errors                          # Array of errorobjects. Empty array if there are no errors.
 
-vda5050_msgs/Info[] informations                     # Array of info-objects. An empty array indicates that the AGV has no information.
+vda5050_msgs/Info[] information                      # Array of info-objects. An empty array indicates that the AGV has no information.
```

Comment:
Hey I believe all changes to this field name must be in the same PR. If we merge this PR as is, it can break stuff such as `vda5050_connector_py/vda5050_controller.py` because it still references `informations` field

Reply:
this shouldnt be here it is my wrong there are 2 separated merge request (informations and convention) I can revert it